robot_framework/eflyt.py

```diff
@@ -133,14 +134,19 @@ def handle_case(browser: webdriver.Chrome, case_number: str, prev_addresses: lis
 
     applicants = get_applicants(browser)
 
+    # Check if all applicants are younger than 19
+    if all(cpr_util.get_age(cpr) < 19 for cpr in applicants):
+        orchestrator_connection.set_queue_element_status(queue_element.id, QueueStatus.DONE, message="Sprunget over: Ingen ansøgere over 18.")
+        return
+
     if room_count >= len(applicants):
         approve_case(browser)
         orchestrator_connection.set_queue_element_status(queue_element.id, QueueStatus.DONE, message="Sag godkendt.")
         return
 
     # Check for parent+child in 1 room
     if room_count == 1 and len(applicants) == 2:
-        is_child = any(get_age(cpr) < 15 for cpr in applicants)
+        is_child = any(cpr_util.get_age(cpr) < 15 for cpr in applicants)
         if is_child:
```

Comment:
This might be a PHP thing, but our coding standards would want this to just be
```
if any(cpr_util.get_age(cpr) < 15 for cpr in applicants):
```

Reply:
It made sense at some point, but you're right